### PR TITLE
improve nnictl import doc

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
add more details for `nnictl import`

无标题

### Checklist ###
  - [x] doc

### CODE REVIEW

File: nni/tools/nnictl/nnictl.py [hunk 1]
```diff
@@ -115,6 +115,11 @@
 For your customized tuner/advisor, the file could have any json content depending on
 how you implement the corresponding methods (e.g., ``import_data``).
 
+.. note::
+
+    You can't see import data on the web portal when you import data to the experiment. Because currently the import data
+    only has the metric and hyper-parameters. But the data in the web portal is a integral trial.
```
Comment (ultmaster) [inline nni/tools/nnictl/nnictl.py:121]: What does `integral trial` mean?

File: nni/tools/nnictl/nnictl.py [hunk 2]
```diff
@@ -115,6 +115,11 @@
 For your customized tuner/advisor, the file could have any json content depending on
 how you implement the corresponding methods (e.g., ``import_data``).
 
+.. note::
+
+    You can't see import data on the web portal when you import data to the experiment. Because currently the import data
```
Comment (ultmaster) [inline nni/tools/nnictl/nnictl.py:120]: see imported data

import data into the experiment

Comment (Lijiaoa) [reply nni/tools/nnictl/nnictl.py:120]: yes! will change it!

File: nni/tools/nnictl/nnictl.py (hunk 1 above)
Comment (Lijiaoa) [reply nni/tools/nnictl/nnictl.py:121]: 完整的trial，拥有id,sequeceid,parameters,final-result...所有字段的trial
用户导入的trial只有超参和final result两个字段

Comment (ultmaster) [reply nni/tools/nnictl/nnictl.py:121]: , while to be visible on the web portal, one trial must have a complete record, including fields like sequence ID, intermediate results and etc.

Comment (Lijiaoa) [reply nni/tools/nnictl/nnictl.py:121]: good suggestions! Thanks very much!

**Comment by Lijiaoa** [comment]
fix yuge's comments and now the page looks like that:
image

**Review by ultmaster — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**